examples/src/main/java/io/github/wechaty/DingDongBot.java

```diff
@@ -12,12 +12,11 @@
 
     public static void main(String[] args) throws Exception {
 
-        String token = "your_token";
+        final String token = System.getenv("WECHATY_PUPPET_HOSTIE_TOKEN");
 
-        if(StringUtils.equals("your_token",token)){
+        if(StringUtils.isBlank(token)){
             throw new Exception("need a token");
```

Comment:
It would be better if we can show a more meaningful message to the user so that they could be able to know how to fix it.

For example, we can make an entry in our README with title `WECHATY_PUPPET_HOSTIE_TOKEN`, and change the error message like the following:

> Error: WECHATY_PUPPET_HOSTIE_TOKEN is not found in the environment variables.  
> You need a TOKEN to run the Java Wechaty. Please goto our README for details:  
> https://github.com/wechaty/java-wechaty-getting-started/#wechaty_puppet_hostie_token

Reply:
ok